frontend/src/components/DeclarationSummary/index.vue

```diff
@@ -20,7 +20,11 @@
       <SummaryModificationButton class="ml-4" v-if="!readonly" @click="router.push(editLink(0))" />
     </h3>
     <div>
-      <SummaryInfoSegment v-if="!!payload.teleicareId" label="Identifiant Teleicare" :value="payload.teleicareId" />
+      <SummaryInfoSegment
+        v-if="!!payload.teleicareDeclarationNumber"
+        label="Identifiant Teleicare"
```

Comment:
```suggestion
        label="Identifiant Téléicare"
```
D'ailleurs ça fera un conflit dans #2133, je le gérerai